src/main/java/com/hm/achievement/utils/RewardParser.java

```diff
@@ -241,6 +255,25 @@ public ItemStack getItemReward(String configAchievement) {
 		return commandReward.split(";[ ]*");
 	}
 
+	/**
+	 * Extracts custom command message from config. Might be null.
+	 *
+	 * @author tassu
+	 * @param configAchievement
+	 * @return the custom command message (null if not present)
+	 */
+	public String getCustomCommandMessage(String configAchievement) {
+		if (!plugin.getPluginConfig().isConfigurationSection(configAchievement + ".Reward.Command")) {
+			return null;
+		}
+
+		if (!plugin.getPluginConfig().contains(configAchievement + ".Reward.Command.Display")) {
```

Comment:
I don't think this `if` block is actually needed, as the `getString` method returns `null` if the value is not found.